lib/cmd/fh2/undeploy.js

```diff
@@ -1,9 +1,9 @@
 /* globals i18n */
 module.exports = undeploy;
```

Comment:
fh2 is no longer maintained. When editing we do not need to change anything in fh2 commands.